src/v2/view-builder/views/ov/ChallengeOktaVerifySSOExtensionView.js

```diff
@@ -0,0 +1,44 @@
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
+import BaseForm from '../../internals/BaseForm';
+import Util from '../../../../util/Util';
+import { loc } from 'okta';
+
+// for BETA,
```

Comment:
Can we change the description here a little bit to state the difference between this file and SSOExtensionView?